src/utils/setConfig.js

```diff
@@ -11,33 +11,25 @@ var stripJsonComments = require( 'strip-json-comments' )
  * 2: user passes location of .stylintrc file to use
  * 3: user has a .stylintrc file in their dir but doesnt pass anything
  * 4: none of the above, fallback to initial config
+ * @param {String} [configpath] If defined, the path to a config-file to read
  * @returns {Function} kick off linter again
 */
-var setConfig = function() {
+var setConfig = function( configpath ) {
```

Comment:
Might be more correct to set it on `this.state` or something like that? As I've mentioned before, I've not touched stampit before, unsure what conventions there are :smile: